packages/multichain/src/scope/constants.ts

```diff
@@ -0,0 +1,47 @@
+import MetaMaskOpenRPCDocument from '@metamask/api-specs';
+import type { KnownCaipNamespace } from '@metamask/utils';
+
+// ScopeString for ecosystems that aren't chain specific
```

Comment:
What's an ecosystem in this context?

Reply:
I think this comment is also misleading. Will rephrase it.

Something like:
`ScopeString for offchain methods that are not specific to a chainId but are specific to a namespace`

Reply:
Changed a few of these comments + cleaned up a duplicated enum here: https://github.com/MetaMask/core/pull/4784/commits/aa24e964a60d94cd7dc9830a90376607c5aafa93

Reply:
Nice! Looks better.